cognite/powerops/client/_generated/afrr_bids/_api_client.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from cognite.client import ClientConfig, CogniteClient, data_modeling as dm
+from cognite.client.credentials import OAuthClientCredentials
+
+from ._api.alert import AlertAPI
+from ._api.bid_document import BidDocumentAPI
+from ._api.bid_method import BidMethodAPI
+from ._api.bid_row import BidRowAPI
+from ._api.price_area import PriceAreaAPI
+from . import data_classes
+
+
+class AFRRBidsAPI:
```

Comment:
We should probably add an instance of this to the `PowerOpsClient` class, like with other API classes.

Reply:
I have updated this now, thanks for the heads up.